samples/ubuntu-datavolume.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: virt.virtink.smartx.com/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: ubuntu-datavolume
+spec:
+  instance:
+    memory:
+      size: 1Gi
+    disks:
+      - name: ubuntu
+      - name: cloud-init
+    interfaces:
+      - name: pod
+  volumes:
+    - name: ubuntu
+      dataVolume:
+        name: ubuntu
+    - name: cloud-init
+      cloudInit:
+        userData: |-
+          #cloud-config
+          password: password
+          chpasswd: { expire: False }
+          ssh_pwauth: True
+  networks:
+    - name: pod
+      pod: {}
+---
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: ubuntu
+spec:
+  source:
+      http:
+        url: https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img 
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    volumeMode: Block
```

Comment:
Block storage is more rare than FS storage in current K8s world. Maybe we should change this sample to use an easy-to-get storage, like Rook NFS?